src/main/java/com/zendesk/maxwell/MaxwellConfig.java

```diff
@@ -270,6 +272,8 @@ protected MaxwellOptionParser buildOptionParser() {
 				.withRequiredArg();
 		parser.accepts( "raft_member_id", "raft memberID.  (may also be specified in raft.xml)" )
 				.withRequiredArg();
+		parser.accepts( "client_max_reconnection_attempts", "client_max_reconnection_attempts" )
+				.withOptionalArg().ofType(Integer.class);
```

Comment:
let's see... 

- let's move this to the "mysql" section of the help.  underneath `--replica_server_id` should do. 
- call it `replication_reconnection_retries`